pip/utils/outdated.py

```diff
@@ -92,14 +93,15 @@ def load_selfcheck_statefile():
         return GlobalSelfCheckState()
 
 
-def pip_version_check(session):
+def pip_version_check(session, options):
     """Check for an update for pip.
 
     Limit the frequency of checks to once per week. State is stored either in
     the active virtualenv or in the user's USER_CACHE_DIR keyed off the prefix
     of the pip script path.
     """
-    installed_version = get_installed_version("pip")
+    pip_pypi_package_name = "pip"
```

Comment:
It's fine to just hardcode ``"pip"`` instead of using a variable. I very much doubt we're ever going to change the name of pip.